lms/templates/courseware/course_updates.html

```diff
@@ -1,5 +1,6 @@
 <%! from django.utils.translation import ugettext as _ %>
-<section>
+<section aria-labelledby="course-updates-heading">
+    <h2 class="hd hd-2 sr" id="course-updates-heading">${_('All course updates')}</h2>
```

Comment:
Update: that's info.html, which is right below. @clrux, where does course_updates.html appear?